### PR TITLE
Lab 8

### PR DESCRIPTION
**Pull request by YaroslavKSE**
Homework

### CODE REVIEW

**Review by tshemsedinov — CHANGES_REQUESTED** [review]
Fix linter errors and warnings

File: lecture_notes/08 collections/6-set.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+ages.add(16)
+ages.remove(20)
+print(ages)
+
+
+def check_if_true(element):
+    if element in ages:
+        return True
+    else:
+        return False
```
[the remark this answers is not in the document]
Comment (YaroslavKSE) [reply lecture_notes/08 collections/6-set.py]: Improved naming, refactored it
P.s see in 'Update 6-set Python'

File: lecture_notes/08 collections/7-dict.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+print(ages)
+ages.pop('Vasia Pupkin', 19)
+print(ages)
+
+
+def check_if_true(element):
+    if element in ages.keys():
+        return True
+    else:
+        return False
```
[the remark this answers is not in the document]
Comment (YaroslavKSE) [reply lecture_notes/08 collections/7-dict.py]: Fixed problems
See in "Update 7-dict Python"

**Comment by YaroslavKSE** [comment]
> Fix linter errors and warnings

Fixed

**Pull request closed without merging**